src/server.coffee

```diff
@@ -591,3 +597,25 @@ else
           type: 'restart-all'
       , 2000
     done()
+
+  # function to return process uptimes
+  exports.getUptime = (callback) ->
+
+    if cluster.isMaster
+      # send reponse back to API request
+      uptime =
+        master: process.uptime()
+      callback null, uptime
+    else
+      # send request to master
+      process.send
+        type: 'get-uptime'
+
+      # listen for response from master
+      process.once 'message', (uptime) ->
+        if uptime.type is 'get-uptime'
```

Comment:
I just thought of something, perhaps we should add an else statement here and call the callback with:

```
{
  master: null
}
```

This would be so that the API call would respond even if it got a message it wasn't expecting.